<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailPageWidget</class>
 <widget class="QWidget" name="DetailPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>513</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>311</width>
    <height>513</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>311</width>
    <height>513</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="detailpageImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1.5px solid black;
image: url(:/detailpageImage/detailpageImage/person_100dp_000000.png);
border-radius: 20px;
background-color: lightgray;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detailpageTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt;</string>
   </property>
   <property name="text">
    <string>연락처 상세</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>291</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="IconLayout">
    <item>
     <widget class="QLabel" name="detailpageCall">
      <property name="styleSheet">
       <string notr="true">color: rgb(89, 133, 225);
image: url(:/detailpageImage/detailpageImage/call_33dp_5985E1.png);
font: 700 7pt;
padding-bottom: 10px;</string>
      </property>
      <property name="text">
       <string>



통화</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="detailpageMessage">
      <property name="styleSheet">
       <string notr="true">color: rgb(89, 133, 225);
image: url(:/detailpageImage/detailpageImage/sms_33dp_5985E1.png);
font: 700 7pt;
padding-bottom: 10px;</string>
      </property>
      <property name="text">
       <string>



메세지</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="detailpagemailImage">
      <property name="styleSheet">
       <string notr="true">color: rgb(89, 133, 225);
font: 700 7pt;
image: url(:/detailpageImage/detailpageImage/mail_33dp_5985E1.png);
padding-bottom: 10px;</string>
      </property>
      <property name="text">
       <string>



메일</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="detailpageInvite">
      <property name="styleSheet">
       <string notr="true">color: rgb(89, 133, 225);
font: 700 7pt;
image: url(:/detailpageImage/detailpageImage/event_33dp_5985E1.png);
padding-bottom: 10px;</string>
      </property>
      <property name="text">
       <string>



약속 초대</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="detailpageNoticeText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(126, 126, 126);
font: 8pt;
padding-left: 10px;</string>
   </property>
   <property name="text">
    <string>※ Notice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detailpagesaveButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/detailpageImage/detailpageImage/box_20dp_999999.png);
background-color: rgb(240, 240, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="detailpageeditButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/detailpageImage/detailpageImage/edit_20dp_999999.png);
background-color: rgb(240, 240, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="detailpageNotice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(126, 126, 126);
font: 8pt;
border: 0px;
padding-left: 10px;</string>
   </property>
   <property name="text">
    <string>메모</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="detailpageCompany">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0,0,0);
border: 0.5px solid gray;
border-radius: 10px;
padding-left: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="detailpageexitButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
image: url(:/detailpageImage/detailpageImage/close_35dp_666666.png);
border: 0px solid black;
border-radius: 15px;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detailpageNoticecompanyText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(126, 126, 126);
background-color: rgb(240, 240, 240);
font: 8pt;
padding-left: 0px;</string>
   </property>
   <property name="text">
    <string>- Company</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="detailpageMail">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0,0,0);
border: 0.5px solid gray;
border-radius: 10px;
padding-left: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detailpageMailtext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(126, 126, 126);
background-color: rgb(240, 240, 240);
font: 8pt;
padding-left: 0px;</string>
   </property>
   <property name="text">
    <string>- Mail</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="detailpageiconOutline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
border: 1px solid rgb(89, 133, 225);
border-radius: 20px;
padding-left: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detailpageLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(126, 126, 126);
font: 7pt;
padding-left: 2px;</string>
   </property>
   <property name="text">
    <string>Edit    Save</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="detailpageaddnewButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/detailpageImage/detailpageImage/person_add_30dp_000000.png);
background-color: rgb(240, 240, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detailpageaddnewLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(126, 126, 126);
font: 8pt;
padding-left: 2px;</string>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="infoFormLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="infoVLayout">
      <item>
       <widget class="QLineEdit" name="detailpageName">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);
font: 12pt;
border: 0px;</string>
        </property>
        <property name="text">
         <string>김진권</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="detailpageNickname">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);
font: 8pt;
color: rgb(180, 180, 180);
border: 0px;</string>
        </property>
        <property name="text">
         <string>(진권이)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="detailpagePosition">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);
font: 8pt;
color: rgb(180, 180, 180);
border: 0px;</string>
        </property>
        <property name="text">
         <string>사원 / 과장</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="detailpagePhone">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);
font: 9pt;
color: rgb(0, 0, 0);
border: 0px;</string>
        </property>
        <property name="text">
         <string>010-1234-5678</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>detailpageLabel</zorder>
  <zorder>detailpageiconOutline</zorder>
  <zorder>detailpageImage</zorder>
  <zorder>detailpageTitle</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>detailpageNoticeText</zorder>
  <zorder>detailpagesaveButton</zorder>
  <zorder>detailpageeditButton</zorder>
  <zorder>detailpageNotice</zorder>
  <zorder>detailpageexitButton</zorder>
  <zorder>label</zorder>
  <zorder>detailpageCompany</zorder>
  <zorder>detailpageNoticecompanyText</zorder>
  <zorder>detailpageMail</zorder>
  <zorder>detailpageMailtext</zorder>
  <zorder>detailpageaddnewButton</zorder>
  <zorder>detailpageaddnewLabel</zorder>
  <zorder>formLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
